Gopkg.lock

```diff
@@ -524,6 +533,12 @@
   packages = ["."]
   revision = "59fac5042749a5afb9af70e813da1dd5474f0167"
 
+[[projects]]
+  name = "github.com/lyft/protoc-gen-validate"
+  packages = ["validate"]
+  revision = "8e6aaf55f4954f1ef9d3ee2e8f5a50e79cc04f8f"
```

Comment:
I think version is enough ( and better ).

Reply:
Oh. I dunno about these. Used whatever dep generated. Kuat?

Reply:
Dunno if @akonradi has some plans for versioning. The import is not used as gogo needs to be implemented there.

Reply:
I'm flattered, but @rodaine is probably the person to talk to. I haven't seen any discussions about versioning in PGV yet, though.

Reply:
There's no hard policy there yet. I typically cut a release when we need to upgrade internally.

Reply:
But is it something we use in pilot, and for what ?

Reply:
I assume we do since why would godeps put a lock for it, otherwise. It might be just an impllicit import somewhere (`import _ ...`).